src/netutils.cpp

```diff
@@ -11,13 +11,25 @@ size_t Curl::write_data_file(char* ptr, size_t size, size_t nmemb, void* stream)
 	return bytes;
 }
 
+constexpr double mBytesProgressUpdateDelta = 0.1; // 100 kB
```

Comment:
Hmm, is 0.1 "MB" equal to 100 kB here? This seems a dumb question, but I just realized that the comment may be wrong.